rsyslog/init.sls

```diff
@@ -55,6 +55,7 @@ rsyslog_mmjsonparse:
 add_adiscon_ppa:
   pkgrepo.managed:
     - name: deb http://ppa.launchpad.net/adiscon/v8-stable/ubuntu focal main
+    - key_url: salt://rsyslog/files/adiscon_ppa_pgp.pub
```

Comment:
can this be a salt path? I would imagine it would need to be a path to a raw file on the minion

Reply:
Yeah I copied this format from [here](https://github.com/OsaroAI/salt-states/blob/master/docker-internal/pkgs.sls#L14). It's the public key so I think it makes sense to just keep it in Salt.